cypress/objects/timeLoggingPage.js

```diff
@@ -0,0 +1,24 @@
+class TimeLoggingPage {
+    
+    visit() {
+        cy.visit('/time-logging'); 
+    }
+
+    getCalendar() {
+        return cy.get('.calendar')
+    }
+
+    getCalendarTodaysDate() {
+        return cy.get('.calendar--selected').find('span').contains(new Date().getDate())
+    }
+
+    getPageTitle() {
+        return cy.get('.page__title')
+    }
+    
+    getUserTitle() {
```

Comment:
This also should be put in some kind of shared file like `common.js`, see above comment